spec/dealership_spec.rb

```diff
@@ -0,0 +1,93 @@
+require './lib/car'
+require './lib/dealership'
+
+
+RSpec.describe Dealership do
```

Comment:
Protip - Can you use a `before(:each)` so you don't need to define dealership, car_1, car_2, etc. in each test?